product_pricelist_simulation/models/pricelist.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2017 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    show_in_simulation = fields.Boolean(string="Show in simulation")
```

Comment:
```suggestion
    show_in_simulation = fields.Boolean(string="Show in simulation", default=True)
```

That option is a great addition to hide some useless pricelist.
However, it should be enabled by default. (User installing that module expect that the feature is enabled).